xbmc/pvr/PVRManager.h

```diff
@@ -70,6 +71,21 @@ namespace PVR
     ManagerStateStarted
   };
 
+  struct PVRManagerEvent
+  {
+    virtual ~PVRManagerEvent() {};
+  };
+
+  namespace PVRManagerEvents
+  {
+    struct Starting    : PVRManagerEvent {};
```

Comment:
i think the idea is that if an event is a struct it can carry aditional data. i do not use this this for pvr manager, but some of the addon events do.